README.md

```diff
@@ -1,2 +1,27 @@
-# exmaples
-Demo and examples to build in Glean
+# Glean project examples
+
```

Comment:
maybe point people at demo.glean.io?  I actually think most of the content from the landing page highlighting usecases could be good here too.  Can do that on another pass too